docs/filesystems/HTTP.md

```diff
@@ -37,3 +37,9 @@ backend {
 
 If there is a need to turn off this `http` filesystem in the default `Local` backend the following Java property
 allows for this: `-Dbackend.providers.Local.config.filesystems.http.enabled=false`.
+
+### Caveats
+
+Using http inputs in Cromwell can produce some unexpected behavior:
+- Files specified by http URIs will be renamed locally, so programs that rely on file extensions or other filenaming conventions may not function properly
+- Files ostensibly located in the same remote http-defined directory will not be colocalted locally. This can cause problems if a program is expecting an index file (e.g. `.fai`), to appear in the same directory as the associated data file (e.g. `.fa`) without offering an means of specifying the index location.
```

Comment:
```suggestion
- Files located in the same remote HTTP-defined directory will not be colocated locally. This can cause problems if a program is expecting an index file (e.g. `.fai`) to appear in the same directory as the associated data file (e.g. `.fa`) without specifying the index location.
```